pennylane/gradients/gradient_transform.py

```diff
@@ -21,34 +21,32 @@
 import pennylane as qml
 from pennylane.measurements import MutualInfoMP, ProbabilityMP, StateMP, VarianceMP, VnEntropyMP
 
-SUPPORTED_GRADIENT_KWARGS = [
+SUPPORTED_GRADIENT_KWARGS = {
     "approx_order",
     "argnum",
+    "atol",
     "aux_wire",
     "broadcast",  # [TODO: This is in param_shift. Unify with use_broadcasting in stoch_pulse_grad
+    "dev",

```

Comment:
`"dev"` also seems like a problematic gradient keyword argument.  

```
qml.QNode(qfunc, device=my_device, dev=some_other_device)
```

Seems extremely confusing.

Reply:
I'd be fine with removing this one again and living with the warning that `param_shift_cv` users keep getting because of it.